tests/languages/sas/options.test

```diff
@@ -0,0 +1,34 @@
+options nodate linesize=72;
+options validmemname=extend validvarname=any;
+----------------------------------------------------
+
+[
+  [
+    "options",
+    [
+      ["options", "options"],
+      ["arg","nodate"],
+      ["arg", "linesize"],
+      ["equals", "="],
+      ["number", "72"],
+      ";"
+    ]
+  ],
+  [
+    "options",
+    [
+      ["options", "options"],
+      ["arg", "validmemname"],
+      ["equals", "="],
+      ["arg", "extend"],

```

Comment:
Isn't it incorrect that `extend` is matched as `arg`?

Reply:
It is. I've added an argValue pattern.